wapitiCore/definitions/htaccess.py

```diff
@@ -24,6 +24,8 @@
 NAME = _("Htaccess Bypass")
 SHORT_NAME = NAME
 
+WSTG_CODE = ["WSTG-CONF-06"]
+
```

Comment:
Please add reference to https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/06-Test_HTTP_Methods